testsuite/features/step_definitions/content_lifecycle_steps.rb

```diff
@@ -0,0 +1,23 @@
+When(/^I click the environment build button$/) do
+  page.find(:xpath, '//*[@id="cm-build-modal-save-button"]').click
+end
+
+When(/^I click promote for dev to qa$/) do
+  page.find(:xpath, '//*[@id="dev_name-promote-modal-link"]').click
+end
+
+When(/^I click promote for qa to prod$/) do
+  page.find(:xpath, '//*[@id="qa_name-promote-modal-link"]').click
+end
+
+When(/^I should see a "([^"]*)" text in the environment "([^"]*)"$/) do |text, env|
```

Comment:
1. Then

2. Preferably no "I" nor "see". `a  "([^"]*)" text should be in the environment "([^"]*)`, or even ` "([^"]*)" should be in the environment "([^"]*)` ?

Reply:
Lots of steps use I in them.